TimberAPI/CustomObjectSystem/CustomObjectCollection.cs

```diff
@@ -11,7 +10,9 @@ public class CustomObjectCollection : IObjectCollection
 
 		public IEnumerable<Object> GetObjects()
 		{
-			return TimberAPI.CustomObjectRegistry.GetAllGameObjects();
+			List<GameObject> objects = TimberAPI.CustomObjectRegistry.GetAllGameObjects();
+			Log.LogInfo($"Loaded {objects.Count} custom objects");
```

Comment:
Getting object should not provoke a loaded message.

Reply:
GetObjects() is called by the game, not users/mods